benchmarks/utils/backend.py

```diff
@@ -25,6 +26,14 @@
     'cpu-ideep',
 ]
 
+_enabled_backend_modes = (
+    os.environ['CHAINER_BENCHMARK_BACKENDS'].split(',')
+    if 'CHAINER_BENCHMARK_BACKENDS' in os.environ
+    else _backend_modes
+)
+
+assert all([x in _backend_modes for x in _enabled_backend_modes])
```

Comment:
How about checking not setting the same mode twice?

Reply:
I think it's better not to check it, as it is harmless. Sometimes flexibility is important for automation. For example I may want to do `CHAINER_BENCHMARK_BACKENDS=cpu,$ADDITIONAL_BACKENDS` to test against arbitrary backends while ensuring that ``cpu`` is always covered.

Reply:
OK, I see.